infrastructure/config/network.template.yml

```diff
@@ -22,11 +22,18 @@ Resources:
         ExecuteApiGatewayEnabled: "Yes"
         AccessLogsCustomBucketNameEnabled: "No"
         AvailabilityZoneCount: 2
-        AllowedDomains: !Join [ ",", [
-          !Sub "cognito-idp.${AWS::Region}.amazonaws.com" ] ]
+        AllowedDomains: !Join [ ",",
+          [ !Sub "cognito-idp.${AWS::Region}.amazonaws.com",
```

Comment:
be careful with this one, as the VPCe for cognito may allow you to route to this domain without having to go over the public internet.